rules/integrations/aws/defense_evasion_route53_dns_query_resolver_config_deletion.toml

```diff
@@ -0,0 +1,103 @@
+description = """
+Identifies when a Route53 Resolver Query Log Configuration is deleted. When a Route53 a Resolver query log configuration
+is deleted, Resolver stops logging DNS queries and responses for the specified configuration. Adversaries may delete
+query log configurations to evade detection or cover their tracks.
+"""
+false_positives = ["Legitimate deletion of Route53 Resolver Query Log Configuration by authorized personnel."]
+from = "now-60m"
+index = ["filebeat-*", "logs-aws.cloudtrail*"]
+interval = "10m"
+language = "kuery"
+license = "Elastic License v2"
+name = "Route53 Resolver Query Log Configuration Deleted"
+note = """
+
+## Triage and Analysis
+
+### Investigating Route53 Resolver Query Log Configuration Deleted
+
+This rule detects when a Route53 Resolver Query Log Configuration is deleted. Deleting these configurations stops the logging of DNS queries and responses, which can significantly impede network monitoring and compromise security visibility. Adversaries may delete these configurations to evade detection, remove evidence, or obscure their activities within a network.
+
+Adversaries target Route53 Resolver query log configurations because these logs can contain evidence of malicious domain queries or responses. By deleting these logs, an adversary can prevent the capture of information that could reveal unauthorized network activities, aiding in avoiding detection and thwarting incident response efforts.
+
+#### Possible Investigation Steps
+
+- **Review the Deletion Details**: Examine the CloudTrail logs to identify when and by whom the deletion was initiated.
+    - Check the `event.action` and `user_identity` elements to understand the scope and authorization of the deletion.
+- **Contextualize with User Actions**: Assess whether the deletion aligns with the user’s role and job responsibilities.
+    - Investigate if similar modifications have occurred recently that could suggest a pattern or broader campaign.
+- **Analyze Access Patterns and Permissions**: Verify whether the user had the appropriate permissions to delete log configurations.
+    - Investigate any recent permission changes that might indicate role abuse or credentials compromise.
+- **Correlate with Other Security Incidents**: Look for related security alerts or incidents that could be connected to the log deletion.
+    - This includes unusual network traffic, alerts from other AWS services, or findings from intrusion detection systems.
+- **Interview the Responsible Team**: If the deletion was initiated by an internal team member, confirm their intent and authorization to ensure it was a legitimate action.
+
+### False Positive Analysis
+
+- **Legitimate Administrative Actions**: Confirm that the deletion was part of scheduled IT operations or network management activities, possibly linked to maintenance or infrastructure updates. Validate this action against change management records or through interviews with relevant personnel.
```

Comment:
New Term rule type is ideal for close to normal behaviors

Reply:
Agreed. If these are deleted TDIR will be hampered significantly though. Often resolver logs are typically sent to S3 buckets.